charts/trino/templates/configmap-coordinator.yaml

```diff
@@ -142,29 +122,26 @@ data:
   {{- end }}
 {{ end }}
 
-{{- range $fileName, $fileContent := .Values.coordinator.additionalConfigFiles }}
+{{- if .Values.accessControl }}
```

Comment:
This doesn't need to be conditional, since the block definition already checks for `.Values.accessControl`